apps/codeforafrica/src/payload/collections/Projects.js

```diff
@@ -0,0 +1,130 @@
+import image from "../fields/image";
+import linkArray from "../fields/links/linkArray";
+import linkGroup from "../fields/links/linkGroup";
+import richText from "../fields/richText";
+import slug from "../fields/slug";
+import tags from "../fields/tags";
+import nestCollectionUnderPage from "../utils/nestCollectionUnderPage";
+
+const Projects = {
+  slug: "projects",
+  admin: {
+    useAsTitle: "name",
+  },
+  fields: [
+    {
+      name: "name",
+      label: {
+        en: "Name",
+      },
+      type: "text",
+      required: true,
+    },
+    image({
+      overrides: {
+        label: {
+          en: "Icon",
+        },
+        name: "icon",
+        required: true,
+      },
+    }),
+    {
+      name: "title",
+      label: { en: "Title" },
+      type: "text",
+      required: true,
+    },
+    richText({
+      name: "subtitle",
+    }),
+    {
+      name: "tagLine",
+      label: {
+        en: "Tag Line",
+      },
+      type: "text",
+    },
+    tags({
+      name: "tag",
+      hasMany: false,
+      admin: {
+        position: "sidebar",
+      },
+    }),
+    linkGroup({ overrides: { name: "externalLink", label: "External Link" } }),
```

Comment:
```suggestion
    linkGroup({ overrides: { name: "externalLink", label: "Link" } }),
```
Shouldn't the label just be Link? To the code, it's external because there is internal link to the page but the content editors, there is just one link, no?